l10n/en/firefox/set-as-default/thanks.ftl

```diff
@@ -27,9 +27,16 @@ set-as-default-thanks-youre-all-set = You’re all set.
 set-as-default-thanks-get-firefox-for-mobile = Get { -brand-name-firefox } for mobile
 set-as-default-thanks-travel-the-internet-with = Travel the internet with protection on all your devices.
 set-as-default-thanks-download-the-app = Download the app
+
+# Obsolete string
 set-as-default-thanks-join-firefox = Join { -brand-name-firefox }
+
+# Obsolete string
 set-as-default-thanks-sign-up-for-a-free-account = Sign up for a free account and sync all your passwords, browsing history, and preferences across your devices.
 
+set-as-default-create-an-account = Create an account
```

Comment:
I checked the brand guidelines and nowhere does it say to use "Join Mozilla" as a call to action for creating an account, so I instead went with this. Let me know if you think it should be named something else

Reply:
The Join Mozilla branding is old. This is great.